modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -620,8 +605,7 @@ Where `accessKey` and `secretKey` are provided by AWS.
 | URI example
 
 | `file:`
-| `URLConnectionSeedProvider` label:deprecated[Deprecated in 5.26], +
```

Comment:
removed?

Reply:
The `URLConnectionSeedProvider` remains in 2025.01 but only supports `http`, `ftp` and `https` now.